### ISSUE TITLE
Make downgrading easier

### ISSUE THREAD

**Issue by rosik** [tarantool/cartridge#1397]
Customer case:

1. Upgrade cartridge 2.5.1 -> 2.6.0
2. Operate the cluster a little bit 
3. Try to downgrade

Result:

```
InitError: section vshard_groups[\"default\"] has unknown parameter \"sched_ref_quota\"
```

**Comment by filonenko-mikhail** [comment]
It seems that this is infrastructure ticket to setup testing process.

**Comment by olegrok** [comment]
Seems that a test was merged but nobody fixed an issue. I reopen this ticket seems there are customers who complains about it.

**Comment by olegrok** [comment]
And seems there are no chances to catch described issue since downgrade test works for 2.7.0. But commit with new vshard option was released in 2.6.0 (https://github.com/tarantool/cartridge/commit/f1dfffc78b82a9089fb8db73346653dda6af4274). So we need to check downgrade to the version below 2.6.0
